csvs_to_sqlite/cli.py

```diff
@@ -151,13 +158,17 @@ def cli(
     # make plural for more readable code:
     extract_columns = extract_column
     del extract_column
+    fixed_columns = fixed_column
```

Comment:
I see why you're doing this - `fixed_columns` is a better variable name since this is a multiple - but you can achieve this in a better way in the option definition like so:

```python

@click.option(
    "fixed_column", # This becomes the parameter name
    "--fixed-column",
    multiple=True,
    help="For colname:value, add a column colname and populate with value",
    default=None,
)
```

Reply:
I was trying to match previous options, but I like these suggestions.  Made the suggested changes.

As is, this will work fine for me, but for flexibility, I think the thing it's missing is allowing for types other than string.

Reply:
That's a good point. Being able to specify that the value should be added as an integer column does seem useful here.

Tricky design challenge though. Here a few possibilities, off the top of my head:

    --fixed-column columnname TEXT mytext
    --fixed-column columnname INTEGER 56
    --fixed-column columnname FLOAT 56.5

So it takes three rather than two. This feels a bit ugly though.

    --fixed-column-text columnname mytext
    --fixed-column-int columnname 56
    --fixed-column-float columnname 56.5

That's three new options instead of one, with the type encoded into the option.

If we assume text is the most common type, we could do this:

    --fixed-column columnname mytext
    --fixed-column-int columnname 56
    --fixed-column-float columnname 56.5

Essentially what you have now, but with extra additional options for setting an integer and setting a float.

I have a slight preference for that last one, but I'm not in love with any of these options. What do you think?

Reply:
Here's what `--help` looks like with that last option:
```
  --fixed-column <TEXT TEXT>...   Populate column with a fixed string
  --fixed-column-int <TEXT INTEGER>...
                                  Populate column with a fixed integer
  --fixed-column-float <TEXT FLOAT>...
                                  Populate column with a fixed float
```
I prototyped it up using this:
```python
@click.option(
    "fixed_columns",
    "--fixed-column",
    type=(str, str),
    multiple=True,
    help="Populate column with a fixed string",
    default=None,
)
@click.option(
    "fixed_columns_int",
    "--fixed-column-int",
    type=(str, int),
    multiple=True,
    help="Populate column with a fixed integer",
    default=None,
)
@click.option(
    "fixed_columns_float",
    "--fixed-column-float",
    type=(str, float),
    multiple=True,
    help="Populate column with a fixed float",
    default=None,
)
```

Reply:
I like the last option as well.  How about date/datetime?  From the code, it looks like `--fixed-column created 20211116 --date created` would work.

I haven't really used Click very much, but I really like the type checking and tests.

Reply:
I integrated the changes above and updated `test_fixed_column` to test all three options.

This didn't work for a date column though:
`--fixed-column created 20211116 --date created`